services/recoveryservices/mgmt/2017-07-01/backup/models.go

```diff
@@ -566,15 +567,15 @@ func PossibleJobStatusValues() []JobStatus {
 }
 
 // JobSupportedAction enumerates the values for job supported action.
-type JobSupportedAction string
+type JobSupportedAction int
 
 const (
 	// JobSupportedActionCancellable ...
-	JobSupportedActionCancellable JobSupportedAction = "Cancellable"
+	JobSupportedActionCancellable JobSupportedAction = iota
 	// JobSupportedActionInvalid ...
-	JobSupportedActionInvalid JobSupportedAction = "Invalid"
+	JobSupportedActionInvalid
 	// JobSupportedActionRetriable ...
-	JobSupportedActionRetriable JobSupportedAction = "Retriable"
+	JobSupportedActionRetriable
```

Comment:
Changes need to confirm from Azure team,  the response from Azure API is like this, 

```
{
  "value": [
    {
      "id": "/Subscriptions/fc51e62f-aef2-49c3-b154-5b313db3f379/resourceGroups/Minghe_test/providers/Microsoft.RecoveryServices/vaults/backupvaultMinghetestdemo/backupJobs/c5529980-4735-4a7b-b360-5073cdef7185",
      "name": "c5529980-4735-4a7b-b360-5073cdef7185",
      "type": "Microsoft.RecoveryServices/vaults/backupJobs",
      "properties": {
        "jobType": "AzureIaaSVMJob",
        "duration": "01:09:58.9495507",
        "actionsInfo": [
          1
        ],
        "virtualMachineVersion": "Compute",
        "entityFriendlyName": "demo",
        "backupManagementType": "AzureIaasVM",
        "operation": "Backup",
        "status": "Completed",
        "startTime": "2019-09-19T03:07:26.0228043Z",
        "endTime": "2019-09-19T04:17:24.972355Z",
        "activityId": "535f4731-0a1b-4dd6-8139-01f66d258ce5"
      }
    }
  ]
}
```

The `actionsInfo` field of a job is a list of number, but there is no document about what a specific number means.